src/styles/main.scss

```
@@ -1,3 +1,3 @@
-body {
-  margin: 0;
-}
+@import "./utils/variables";
```

```suggestion
@import "./utils/_variables";
```

your file in the units folder is named differently

---

You can write without "_" utility files. This is an example: https://mate.academy/learn/css-advanced/sass#/video/932

---

agree, leave it as it is